charter.md

```diff
@@ -1,134 +1,159 @@
 # Charter of Data Distribution Working Group (DDWG)
 
+This charter outlines the necessary information that the DDWG requires to operate effectively.
+
+## Updates
+| Date | Changes |
+| --- | --- |
+| November 2013 | Initial Charter creation |
+| October 2018 | Hard drives no longer distributed regularly, BAS now distribution institution | 
+| May 2020 | Members list updated, mailing list change, definitions added,  |
+
+## Definitions
+ - IQDAT/BFIQ: Beamformed IQ data
+ - RAWACF/DAT: Correlation data
+ - Level 0 Data: IQDAT or BFIQ data
+ - Level 1 Data: RAWACF or DAT data
+ - WG: Working group
+ - PI Institutions: Institutions that produce SuperDARN radar data adhering to the PI agreement
+ - DSWG: Data Standards Working Group: <https://github.com/SuperDARN/dswg-published-docs>
+ - DAWG: Data Analysis Working Group: 
+ 
+## Institutions
+ - BAS: British Antarctic Survey
+ - USASK: University of Saskatchewan
+ - VT : Virginia Tech
+ - NIPR: National Institute of Polar Research
+ - JHU/APL: Johns Hopkins University Applied Physics Laboratory
+ - UAF: University of Alaska Fairbanks
+ - IAPS/INAF: Institute for Space Astrophysics and Planetology 
+ - BOM: Bureau of Meteorology
+ - PRIC: Polar Research Institute of China
+ - ISEE: Institute for Space-Earth Environmental Research
+ - IRAP: Research Institute in Astrophysics and Planetology 
+ - NICT: National Institute of Information and Communications Technology
+ - DICE/UEC: University of Electro-Communications
+ - SDU: Shandong University
+ - UNIS: The University Centre in Svalbard
+ - UKZN: University of KwaZulu-Natal
+ 
 ## Purpose
 
-This working group (WG) is charged with maintaining the protocols for transferring RAWACF and DAT 
-files from the PI institutions, first to a holding institution (Virginia Tech) and ultimately to
-a distribution institution (University of Saskatchewan) where they are merged and copied onto hard 
-drives for distribution. The cadence for distribution of the drives is every four months 
-(Jan-Apr, May-August, September-December). This working group was created after the SuperDARN 
-workshop in Moose Jaw, Saskatchewan in 2013. The group is also charged with maintaining an 
-augmentation to the hard drive distribution, any networked SuperDARN Data Mirrors.
+This WG is charged with maintaining the protocols for transferring Level 1 data 
+files from PI institutions, first to holding institutions and 
+ultimately to distribution institutions where they are merged, checked, catalogued and copied to 
+servers for community access. 
```

Comment:
what do we mean by 'community' access? Should this mean scientific community?  SuperDARN community? 'Anyone who uses SuperDARN data' community?

Reply:
DAWG states that users are "scientists and students"

Reply:
Not sure if this is where @mts299 was referencing, but the current DAWG charter (and web info) is in the RST repo's wiki: https://github.com/SuperDARN/rst/wiki .  We're similarly updating our charter and will post the revised version there (file name shouldn't change) once approved by the PIs.